store/book-api.ts

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { Module, VuexModule,Mutation, Action } from 'vuex-module-decorators'
+
+import axios from 'axios'
+
+import { saveAuthSession, clearAuthSession, loadAuthSession} from '~/utils/auth'
+import { API_POST_AUTHORIZE } from '~/constant/api'
+
+@Module({
+  name: 'book-api',
+  stateFactory: true,
+  namespaced: true,
+})
+export default class BookAPI extends VuexModule {
+  token = ''
+  sessionWallet = ''
+  isRestoringSession = false
+
+  @Mutation
+  setToken(inputToken: string) {
+    this.token = inputToken
+  }
+
+  @Mutation
+  setSessionWallet(inputWallet: string) {
+    this.sessionWallet = inputWallet
+  }
+
+  @Mutation
+  setIsRestoringSession(inputIsRestoringSession: boolean) {
+    this.isRestoringSession = inputIsRestoringSession
+  }
+
+  @Action
+  async authenticate({inputWallet = '', signature = {}}: {inputWallet?: string, signature?: any}) {
+    try {
+      const { data } = await axios.post(
+        API_POST_AUTHORIZE,
+        {
+          expiresIn: '7d',
```

Comment:
Why set `expiresIn` here when it is also set in `signMessageMemo` ?

Reply:
[🎨 Remove duplicate expiresIn](https://github.com/likecoin/app-like-co/pull/474/commits/1015cedfc26cdcf0cd51c3bd319f4994d2b0e3a1)